Instruments/config/2016/Skimmer.cfg

```diff
@@ -71,221 +68,162 @@ file: DYJetsToLL_M-50_HT-600to800.root
-file: WJetsToLNu_HT-1200To2500.root
-file: WJetsToLNu_HT-1200To2500_ext1.root
-file: WJetsToLNu_HT-2500ToInf.root
-file: WJetsToLNu_HT-2500ToInf_ext1.root
-
-[WW]
-merged_output: WW.root
-file_xs: 50.00 WWToLNuQQ.root WWToLNuQQ_ext1.root
-file_xs: 12.18 WWTo2L2Nu.root
-file_xs: 51.72 WWTo4Q.root
-#file_xs: 50.00 WWTo1L1Nu2Q.root
-
-[WZ]
-merged_output: WZ.root
-#file_xs: 4.43 WZTo3LNu_amcatnloFXFX.root
-file_xs: 4.43 WZTo3LNu.root WZTo3LNu_ext1.root
-#file_xs: 47.13 WZ_TuneCUETP8M1.root WZ_TuneCUETP8M1_ext1.root
-file_xs: 10.71 WZTo1L1Nu2Q.root
-file_xs: 5.60 WZTo2L2Q.root
-file_xs: 3.03 WZTo1L3Nu.root
+file: WJetsToLNu_HT-100To200.root WJetsToLNu_HT-100To200_ext1.root WJetsToLNu_HT-100To200_ext2.root
+file: WJetsToLNu_HT-200To400.root WJetsToLNu_HT-200To400_ext1.root WJetsToLNu_HT-200To400_ext2.root
+file: WJetsToLNu_HT-400To600.root WJetsToLNu_HT-400To600_ext1.root
+file: WJetsToLNu_HT-600To800.root WJetsToLNu_HT-600To800_ext1.root
+file: WJetsToLNu_HT-800To1200_ext1.root WJetsToLNu_HT-800To1200.root
+file: WJetsToLNu_HT-1200To2500.root WJetsToLNu_HT-1200To2500_ext1.root
+file: WJetsToLNu_HT-2500ToInf.root WJetsToLNu_HT-2500ToInf_ext1.root
+file: WJetsToLNu.root WJetsToLNu_ext2.root
+file: W1JetsToLNu.root
+file: W2JetsToLNu.root  W2JetsToLNu_ext1.root
+file: W3JetsToLNu.root W3JetsToLNu_ext1.root
+file: W4JetsToLNu.root W4JetsToLNu_ext1.root W4JetsToLNu_ext2.root
+
+[VV]
+merged_output: VV.root
+file_xs: 43.53 WWToLNuQQ.root WWToLNuQQ_ext1.root
+file_xs: 10.48 WWTo2L2Nu.root
+file_xs: 45.2 WWTo4Q.root
+file_xs: 80.74 WWTo1L1Nu2Q_amcatnloFXFX.root
+file_xs: 4.679 WZTo3LNu-amcatnloFXFX.root
+file_xs: 0.0 WZTo3LNu.root WZTo3LNu_ext1.root
+file_xs: 23.43 WZ.root WZ_ext1.root
+file_xs: 11.66 WZTo1L1Nu2Q_amcatnloFXFX.root
+file_xs: 3.054 WZTo1L3Nu_amcatnloFXFX.root
+file_xs: 6.331 WZTo2L2Q_amcatnloFXFX.root
+file_xs: 3.688 ZZTo2L2Q_amcatnloFXFX.root
+file_xs: 3.688 ZZTo2L2Q.root
+file_xs: 4.033 ZZTo2Q2Nu_amcatnloFXFX.root
+file_xs: 0.564 ZZTo2L2Nu.root ZZTo2L2Nu_ext1.root
+file_xs: 1.204 ZZTo4L-amcatnloFXFX_ext1.root
+file_xs: 1.26 ZZTo4L.root ZZTo4L_ext.root
+file_xs: 10.16 ZZ.root ZZ_ext1.root
+file_xs: 0.0 ZZTo2Q2Nu.root
 
-[ZZ]
-merged_output: ZZ.root
-file_xs: 3.22 ZZTo2L2Q_amcatnloFXFX.root
-#file_xs: 3.22 ZZTo2L2Q.root
-file_xs: 4.04 ZZTo2Q2Nu_amcatnloFXFX.root
-#file_xs: 4.04 ZZTo2Q2Nu.root
-file_xs: 7.06 ZZTo4Q_amcatnloFXFX.root
-file_xs: 0.56 ZZTo2L2Nu.root ZZTo2L2Nu_ext1.root
-file_xs: 1.21 ZZTo4L_13TeV-amcatnloFXFX-pythia8_ext1.root
-#file_xs: 1.21 ZZTo4L.root ZZTo4L_ext1.root
-#file_xs: 16.52 ZZ_TuneCUETP8M1.root ZZ_TuneCUETP8M1_ext1.root
 
 [VVV]
 merged_output: VVV.root
-file_xs: 0.209 WWW_4F.root
-file_xs: 0.165 WWZ.root
-file_xs: 0.056 WZZ.root
-file_xs: 0.014 ZZZ.root
+file_xs: 0.2086	 WWW_4F-amcatnlo.root
+file_xs: 0.1651  WWZ-amcatnlo.root
+file_xs: 0.0556  WZZ-amcatnlo.root
+file_xs: 0.0140 ZZZ-amcatnlo.root
+
+[VH]
+merged_output: VH.root
+file_xs: 0.07814 ZH_HToBB_ZToLL_M125_amcatnloFXFX.root
```

Comment:
it is the same final state as the line below but has a different generator. Comment this one.